encrypted-media-respec.html

```diff
@@ -2331,15 +2365,16 @@
 
         <section id="queue-waitingforkey">
           <h4>Queue a "waitingforkey" Event</h4>
-          <p>The Queue a "waitingforkey" Event algorithm is run when the CDM cannot decrypt the current block.
+          <p>The Queue a "waitingforkey" Event algorithm is run when the CDM cannot decrypt an encrypted block.
             It should only be called when the <a def-id="htmlmediaelement"></a> object is <a def-id="potentially-playing"></a> and its <a def-id="readystate"></a> is equal to <a def-id="have-future-data"></a> or greater.  
```

Comment:
I'm not sure this statement is correct anymore. I think we allow it to be called when `<var>waiting for key</var>` is true (there is an early exit below), which would mean it is blocked, which means it is not potentially playing.

If we want to ensure `<var>waiting for key</var>` is not true when this is called (makes sense given the name of the algorithm) then this statement would be correct.